src/dependency/Dependency/ServiceProviderExtensions/Extensions.GetOrThrow.cs

```diff
@@ -24,7 +24,7 @@ partial class ServiceProviderExtensions
                 return (T)service;
             }
 
-            throw new InvalidOperationException($"Service {typeof(T)} can not be resolved by service provider.");
+            throw new InvalidOperationException($"A service of the type {typeof(T)} can not be resolved by the service provider.");
```

Comment:
It seems '`the type {typeof(T)}`' is redundant and '`type {typeof(T)}`' is enough

Reply:
Have done